nf_core/pipeline-template/README.md

```diff
@@ -25,57 +25,76 @@
+```
 
-   > - The pipeline comes with config profiles called `docker`, `singularity`, `podman`, `shifter`, `charliecloud` and `conda` which instruct the pipeline to use the named tool for software management. For example, `-profile test,docker`.
-   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
-   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
-   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
+Each row represents a fastq file (single-end) or a pair of fastq files (paired end).
 
-4. Start running your own analysis!
+Now, you can run the pipeline using:
 
-   <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
+<!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
 
-   ```bash
-   nextflow run {{ name }} --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
-   ```
+```bash
+nextflow run {{ name }} \
+   --input samplesheet.csv \
+   -profile <docker/singularity/.../institute>
+```
+
+> **Warning**
+> Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those
+> provided by the `-c` Nextflow option can be used to provide any configuration **except for parameters**;
+> see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
+
+{% if branded -%}
+
+For more details, please refer to the [usage documentation](https://nf-co.re/{{ short_name }}/usage) and the [parameter documentation](https://nf-co.re/{{ short_name }}/parameters).
+
+{% endif -%}
 
 {% if branded -%}
 
-## Documentation
+## Pipeline output
 
-The {{ name }} pipeline comes with documentation about the pipeline [usage](https://nf-co.re/{{ short_name }}/usage), [parameters](https://nf-co.re/{{ short_name }}/parameters) and [output](https://nf-co.re/{{ short_name }}/output).
+[results](https://nf-co.re/{{ short_name }}/results) section of the documentation.
```

Comment:
```suggestion
To see the the results of a test run with a full size dataset refer to the [results](https://nf-co.re/{{ short_name }}/results) section of the documentation.
```

Reply:
good catch, thank you! Seems something got lost in the last commit :/